stdlib/public/Concurrency/Clock.swift

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+
+@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, macCatalyst 9999, *)
```

Comment:
Nit: Please use our [availability macros](https://github.com/apple/swift/blob/main/docs/StandardLibraryProgrammersManual.md#availability) here (and in similar additions throughout this PR) to help prevent future clerical errors.

```suggestion
@available(SwiftStdlib 9999, *)
```